src/registry.json

```diff
@@ -1414,6 +1414,35 @@
           "checksum": "YeL3PfCT8hjTTHED2AD0BvsbkjNEjmC3pLxzjgXLFqE="
         }
       }
+    },
+    "npm:@chainsafe/polkadot-snap": {
+      "id": "npm:@chainsafe/polkadot-snap",
+      "metadata": {
+        "name": "Polkadot Wallet",
+        "author": {
+          "name": "ChainSafe",
+          "website": "https://chainsafe.io/"
+        },
+        "website": "https://metamask-snap-polkadot.pages.dev/",
```

Comment:
This goes to a demo page

Reply:
This is the reference website for the Snap. It is called a demo but it's the best option until other dapps implement the Snap.

Reply:
If we're expecting other websites to implement this Snap, I'd rather just omit this field. Keep in mind that we're sending users here after installation. I think right now it would be a confusing experience.

Reply:
OK I have removed the field for now and will ask ChainSafe to modify the site before adding.

Reply:
For reference we are not expecting other websites to implement this Snap. This is an open platform, each Snap dev is responsible for their Snap. I hope they expect websites to implement their Snap if applicable.